os/migrating-to-clcs.md

```diff
@@ -0,0 +1,377 @@
+# Migrating from Cloud Configs to Container Linux Configs
```

Comment:
I'd make these both singular. Also, is usually it would be "cloud-config", IIRC, no caps